releasetasks/templates/final_verify.yml.tmpl

```diff
@@ -0,0 +1,49 @@
+{% set buildername = "{}_final_verify".format(branch) %}
+-
+    taskId: "{{ stableSlugId(buildername) }}"
+    reruns: 5
+    task:
+        provisionerId: aws-provisioner-v1
+        workerType: b2gtest
+        created: "{{ now }}"
+        deadline: "{{ now.replace(hours=6) }}"
+        expires: "{{ never }}"
+        priority: "high"
+        retries: 5
+        scopes:
+
+        payload:
+            maxRunTime: 7200
+            image: rail/python-test-runner
+            cache:
+                
+            command:
+                - /bin/bash
+                - -c
+                - hg clone https://hg.mozilla.org/build/tools && cd tools && hg up -r $TAG && cd release/ && ./final-verification.sh $RELEASE_CONFIGS"
```

Comment:
In the future, need to update the image to include tc-vcs, so we can simplify this.